### ISSUE TITLE
Possible to filter out and exclude Ifname?

### ISSUE THREAD

**Issue by Thinkgreenn** [influxdata/telegraf#4436]
Hello All,

I'm not sure this is the correct area to post htis.
I was wondering if there is a possibility to filter and exclude ifname?

Example screenshot:
https://i.imgur.com/wl9P3uO.png
Includes all the instances 

But i only want to selectively see 2 of them

**Comment by glinton** [comment]
This may be what you are looking for: [processor configuration](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#processor-configuration)

**Comment by Thinkgreenn** [comment]
Thank you very much, does this also apply for snmp settings

**Comment by danielnelson** [comment]
@Thinkgreenn Yes, you can use the measurement filtering options with any plugin.

**Comment by Thinkgreenn** [comment]
Thank you so much, means a lot to me. I will try to figure it out
